Azure-Samples/Custom-vision-service-iot-edge-raspberry-pi
# Enable image stream via WebSocket

## Purpose

Enable image stream via Websocket.

## Does this introduce a breaking change?

```
[X] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[X] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

1. Deploy modules to device via Docker/Iotedge.
2. Open a browser to "localhost:5012"
3. A image/video stream should show up.

## What to Check
Verify that the following are valid
* ...

## Other Information
Stream through web browser on Raspberry Pi 3 might be slow,
